### ISSUE TITLE
🐛 [BUG] - Links in Prose Component that break into new lines do not meet visual a11y requirements

### ISSUE THREAD

**Issue by rfearing** [primer/brand#950]
### Describe the bug

Context: 
- In Prose components, when a link breaks into a new line, it does not show an underline and relies on color only to indicate a link.
- You can see on this Social Impact page that there are multiple links and one (that is broken into a new line) does not have the underline. 
- [WCAG 1.4.1 Use of Color](https://www.w3.org/TR/WCAG21/#use-of-color)
- [Related accessibility audit issue](https://github.com/github/accessibility-audits/issues/10581)

Cause: An absolutely positioned element is used for a visual underline alongside `text-decoration: none`

### Reproduction steps

```bash
1. Go to https://github.com/social-impact/insights/perspectives/11-14-2024-social-impact-at-universe-2024
2. View "GitHub Universe" appears as expected.
3. View "The Hive, USA for UNCHR" does not appear as expected.
```

### Expected behavior

```bash
Both links should appear with an underline.
```

### Screenshots

Video below: adding a `br` tag shows the underline as expected.

https://github.com/user-attachments/assets/a4f97026-4eaf-4f2e-b5fa-b54ab691a60d

<img width="866" alt="Some links include underlines while others do not." src="https://github.com/user-attachments/assets/178276a0-bdde-4243-99fe-f79c60ec59c3" />

### Browsers

Chrome

### OS

Mac

**Comment by rezrah** [comment]
@rfearing thanks for reporting this issue. The underline was missing because we were previously using a border instead of text-decoration. This was fixed in [a recent release](https://github.com/primer/brand/releases/tag/%40primer%2Freact-brand%400.48.0) ([PR](https://github.com/primer/brand/pull/945)), so I think you'll just need a package update to resolve your issue. 

[Here's a Storybook link](https://stunning-chainsaw-j82glqz.pages.github.io/?path=/story/components-prose--playground&args=enableFullWidth:!true) where you can verify the fix in the aforementioned release. Video below too.

https://github.com/user-attachments/assets/a8d52f8c-c26a-4660-bf10-01b3fdd5a98e

Going to close this for now, but feel free to reopen if needed.
